services/app-web/src/pages/StateSubmission/updateSubmissionTransform.ts

```diff
@@ -37,34 +38,66 @@ function cleanDraftSubmission(
 // There's probably some Typescript Cleverness™ we could do for this mapping function
 // but for now the compiler complains if you forget anything so ¯\_(ツ)_/¯
 function updatesFromSubmission(draft: DraftSubmission): DraftSubmissionUpdates {
+    console.log('draft', draft)
     return {
         programIDs: draft.programIDs,
         submissionType: draft.submissionType,
         submissionDescription: draft.submissionDescription,
-        documents: stripTypename(draft.documents),
```

Comment:
🤔  I think this might be the cause of my issues. I need to manually put back the type name everywhere its needed (and remove this code here) to get GQL recognizing my faked domain model > GQL submission